README.md

```diff
@@ -89,3 +92,12 @@ for additional information.
   $ mkdir /some/dir/nexus-data && chown -R 200 /some/dir/nexus-data
   $ docker run -d -p 8081:8081 --name nexus -v /some/dir/nexus-data:/nexus-data sonatype/docker-nexus3
   ```
+
+## Getting help
+
+Looking to contribute to our Docker image but need some help? There's a few ways to get information or our attention:
+
+* File a public issue here
```

Comment:
> I could link to the actual page, WDYT?

+1, I think that provides more clarity, potentially in addition to rephrasing to "here on Github"